testah-junit/src/main/java/org/testah/runner/httpLoad/HttpActor.java

```diff
@@ -83,9 +89,8 @@ public ActorRef getWorkerRouter() {
         return results;
     }
 
-    public static HashMap<Long, List<ResponseDto>> resetResults() {
+    public static void resetResults() {
         results = new HashMap<Long, List<ResponseDto>>();
-        return results;
     }
```

Comment:
Findbug does complain that lines 120 and 166 int HttpAkkaRunner ( i.e. HttpActor.resetResults(); ) ignore the returned value of resetResults. The returned value is used nowhere, might as return void.